packages/fxa-admin-panel/README.md

```diff
@@ -2,6 +2,20 @@
 
 This is an internal resource for FxA Admins to access a set of convenience tools.
 
+## Development
+
+- `npm run start|stop|restart` to start, stop, and restart the server as a PM2 process
+- `npm run build` to create a production build
+
+**External imports**
+
+You can import React components into this project. This is currently restricted to `fxa-components`:
+
+```javascript
+// e.g. from within a file in ./src, assuming the HelloWorld exists
+import HelloWorld from '../../fxa-components/HelloWorld';
```

Comment:
Did you look further into aliased imports? That `import HelloWorld from '@fxa-components/HelloWorld'` example you had before would be _sick_, but I know there were issues around TS?

Reply:
I did, yeah - I [got it working](https://github.com/jodyheavener/rescripts-react-app/tree/jh/resolve-shared) to the extent that Webpack compiles, but the CRA output complains about modifying TSConfig paths, and I can't quite get TypeScript to acknowledge the custom module.

Reply:
Here's this branch **with** shared component resolution, if you want to see what I mean: https://github.com/mozilla/fxa/compare/jh/4817-linkexternal-admin...jh/4817-linkexternal-admin-resolved-modules?expand=1

Reply:
Might be good to have @dannycoates look this over and also that `"pretest:frontend": "cd ../fxa-components && npm install",` bit, I would argue it might be worth investigating here to see if getting those aliased imports working now because it'd be _such_ a handy convenience feature to use from now on, but also not worth spending the entire day on since we can implement it later. 🤔

Reply:
Thanks to @dannycoates for the help sorting out the custom module stuff.

Reply:
Also, I'm going to switch the `pretest` thing to only installing component deps in CI builds (`test-ci.sh`), so we're not slowing down development testing.